Neos.ContentRepositoryRegistry/Configuration/Settings.yaml

```diff
@@ -105,6 +105,16 @@ Neos:
         #   'My.Package:SomeCommandHook': # just a name
         #     factoryObjectName: My\Package\CommandHook\SomeCommandHookFactory
 
+        # PERFORMANCE TRACING (Local Debugging)
+        # To enable Performance Tracing, comment-in the following lines:
+        #debug:
+        #  tracer:
```

Comment:
I'd suggest to rename this to performanceTracer.

should we keep the "debug" key or not?